app/services/hyrax/admin_set_service.rb

```diff
@@ -64,5 +66,17 @@ def data_attributes(admin_set)
         attrs['data-visibility'] = permission_template.visibility unless permission_template.visibility.blank?
         attrs
       end
+
+      # Count number of files from admin set works
+      # @param [Array] Solr search result
```

Comment:
Add a `@return` YARD comment that explains it returns a hash with id as keys and counts as values.